### PR TITLE
Update security policy

### PR DESCRIPTION
**Pull request by ashie**
**Which issue(s) this PR fixes**: 
N/A

**What this PR does / why we need it**: 
We've enabled `Private vulnerability reporting` feature of GitHub: https://docs.github.com/en/code-security/security-advisories/guidance-on-reporting-and-writing/privately-reporting-a-security-vulnerability

Using it is desired instead of sending email to each developers.

**Docs Changes**:
N/A

**Release Note**: 
N/A

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -4,15 +4,11 @@
 
 | Version   | Supported          |
 | -------   | ------------------ |
-| 1.14.x    | :white_check_mark: |
-| <= 1.13.x | :x:                |
+| 1.16.x    | :white_check_mark: |
+| 1.15.x    | :white_check_mark: |
```
Comment (ashie) [inline SECURITY.md:8]: I'm planning to release fluentd v1.15.4 & td-agent v4.4.3 with some backport patches, so 1.15.x is still supported.

Comment (cosmo0920) [reply SECURITY.md:8]: Can we add documentation for supporting strategies/periods for multiple versions?
To stabilize maintenance process is needed, I guess.

e.g.)
When the new version v1.16 is released, the previous series should be supported after 6 months later or something like that.

Comment (kenhys) [reply SECURITY.md:8]: Yes, it should be clarified later. Anyway, main scope of this PR is updating contact address.

**Review by kenhys — APPROVED** [review]
LGTM.

File: SECURITY.md (hunk 1 above)
Comment (ashie) [reply SECURITY.md:8]: We'll clarify it after LTS release is on rail.

**Pull request merged**